dlpy/applications/efficientnet.py

```diff
@@ -0,0 +1,292 @@
+
+    Parameters
+    ----------
+    conn : CAS
+        Specifies the CAS connection object.
+    model_table : string or dict or CAS table, optional
+        Specifies the CAS table to store the deep learning model.
+    n_classes : int, optional
+        Specifies the number of classes. If None is assigned, the model will
+        automatically detect the number of classes based on the training set.
+        Default: 1000
+    n_channels : int, optional
+        Specifies the number of the channels (i.e., depth) of the input layer.
+        Default: 3
+    width : int, optional
+        Specifies the width of the input layer.
+        Default: 224
+    height : int, optional
+        Specifies the height of the input layer.
+        Default: 224
+    width_coefficient: float,
+        Specifies the scale coefficient for network width.
+    depth_coefficient: float,
+        Specifies the scale coefficient for network depth.
+    dropout_rate: double, optional
+        Specifies the dropout rate before final classifier layer.
+    drop_connect_rate: float,
+        Specifies the dropout rate at skip connections.
+    depth_divisor: integer,
+        Specifies the unit of network width.
+    activation_fn: string, optional
+        Specifies the activation function
+    blocks_args: list of dicts
+         Specifies parameters to construct blocks for the efficientnet model.
+    offsets : double or iter-of-doubles, optional
+        Specifies an offset for each channel in the input data. The final input
+        data is set after applying scaling and subtracting the specified offsets.
+        Default: (255*0.406, 255*0.456, 255*0.485)
+    norm_stds : double or iter-of-doubles, optional
+        Specifies a standard deviation for each channel in the input data.
+        The final input data is normalized with specified means and standard deviations.
+        Default: (255*0.225, 255*0.224, 255*0.229)
+    random_flip : string, optional
+        Specifies how to flip the data in the input layer when image data is
+        used. Approximately half of the input data is subject to flipping.
+        Valid Values: 'h', 'hv', 'v', 'none'
+    random_crop : string, optional
+        Specifies how to crop the data in the input layer when image data is
+        used. Images are cropped to the values that are specified in the width
+        and height parameters. Only the images with one or both dimensions
+        that are larger than those sizes are cropped.
+        Valid Values: 'none', 'unique', 'randomresized', 'resizethencrop'
+    random_mutation : string, optional
+        Specifies how to apply data augmentations/mutations to the data in the input layer.
+        Valid Values: 'none', 'random'
+
+
+    Returns
+    -------
+    :class:`Model`
+
+    References
+    ----------
+    https://arxiv.org/pdf/1905.11946.pdf
+
+    '''
+
+    def _make_divisible(v, divisor, min_value=None):
+        # make number of channel divisible
```

Comment:
Even though these are internal methods, they should have docstrings indicating what the parameters are and the types.

Reply:
fixed